### PR TITLE
CONCF-422 | Create background logic for breakpoints

### PR DESCRIPTION
**Pull request by vforge**
Ping @bognix @dianafa @nandy-andy for CR

### CODE REVIEW

File: skins/oasis/css/core/breakpoints-background.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+@if unitless($background-height) {
+	$background-height : $background-height * 1px;
+}
+@if unitless($background-width) {
+	$background-width : $background-width * 1px;
+}
+$width-gradient: 20px;
+
+$background-width-half: $background-width / 2;
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: you don't need to round the value when you divide by 2?

**Comment by dianafa** [comment]
LGTM

File: skins/oasis/css/core/breakpoints-background.scss (hunk 1 above)
Comment (vforge) [inline skins/oasis/css/core/breakpoints-background.scss]: It's not needed - browsers are fine with 0.5px - which should be really good if browser supports retina properly.
Similar code is in https://github.com/Wikia/app/blob/dev/skins/oasis/css/core/background.scss and worked fine for at least a year. 
Do you still want to change it?

File: extensions/wikia/AdEngine/js/template/skin.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ define('ext.wikia.adEngine.template.skin', [
 				pixelElement,
 				pixelUrl;
 
-			if (win.wgOasisResponsive || win.skin === 'venus') {
+			if (win.wgOasisResponsive || win.wgOasisBreakpoints || win.skin === 'venus') {
```
Comment (vforge) [inline extensions/wikia/AdEngine/js/template/skin.js]: Pinging @gabrys for this line.

Comment (gabrys) [inline extensions/wikia/AdEngine/js/template/skin.js]: On 04/13/2015 04:18 PM, V. wrote:
> In extensions/wikia/AdEngine/js/template/skin.js
> <https://github.com/Wikia/app/pull/6918#discussion_r28241191>:
>
>> @@ -33,7 +33,7 @@ define('ext.wikia.adEngine.template.skin', [
>>  				pixelElement,
>>  				pixelUrl;
>>
>> -			if (win.wgOasisResponsive || win.skin === 'venus') {
>> +			if (win.wgOasisResponsive || win.wgOasisBreakpoints || win.skin === 'venus') {
>
> Pinging @gabrys <https://github.com/gabrys> for this line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6918/files#r28241191>.
>
The line looks OK

File: skins/oasis/css/core/breakpoints-background.scss (hunk 1 above)
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: no, you convinced me :)

File: skins/oasis/css/core/breakpoints-background.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+			&:after,
+			&:before {
+				width: $width-gradient;
+			}
+
+			&:after {
+				background-color: rgba( $color-body-middle, 0 );
+				background-image: -webkit-linear-gradient(right, rgba( $color-body-middle, 0 ) 0%, $color-body-middle 100%);
+				background-image: linear-gradient(to left, rgba( $color-body-middle, 0 ) 0%, $color-body-middle 100%);
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: 1. Check if we have mixin for gradients.
2. You can move :after and :before to mixin and pass to it either "left" or "right"

**Comment by bognix** [comment]
LGTM

**Comment by hakubo** [comment]
Ping: @PiotrGackowski @gabrys - how does it loook?

**Comment by gabrys** [comment]
Code looks OK to me.

**Comment by gabrys** [comment]
For QA there's https://wikia-inc.atlassian.net/browse/ADEN-1937

**Pull request merged**